File: backend/logistics-users-service/src/main/java/com/tdtu/logistics_users_service/dto/request/CreateReceiverRequest.java
package com.tdtu.logistics_users_service.dto.request;

import lombok.*;
import lombok.experimental.FieldDefaults;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class CreateReceiverRequest {

    String customerId; // Id của khách hàng.

    String fullName; // Tên đầy đủ của người nhận.

    String phoneNumber; // Số điện thoại của người nhận.

    String email; // Email liên hệ của người nhận.

    String province; // Tỉnh.

    String district; // Huyện.

    String ward; // Xã/Phường.

    String street; // Đường.

    String postalCode; // Mã bưu chính.
}

File: my-app/src/modules/order/models/CreateReceiverRequest.ts
export interface CreateReceiverRequest {
    customerId: string; // Id của khách hàng.

    fullName: string; // Tên đầy đủ của người nhận.

    phoneNumber: string; // Số điện thoại của người nhận.

    email: string; // Email liên hệ của người nhận.

    province: string; // Tỉnh.

    district: string; // Huyện.

    ward: string; // Xã/Phường.

    street: string; // Đường.

    postalCode: string; // Mã bưu chính.
}